octoprint_rtmpstreamer/__init__.py

```diff
@@ -436,6 +444,17 @@ def _build_overlay(self):
             elif overlay_style == "wm_tl":
                 img.paste(watermark, (padding, padding))
 
+        if 'prusaslicerthumbnails' in self._plugin_manager.plugins and self._settings.get(["include_thumb"]):
+            if os.path.exists(os.path.join(self._settings.global_get_basefolder("data"), "prusaslicerthumbnails", filename + ".png")):
+                thumb = Image.open(os.path.join(self._settings.global_get_basefolder("data"), "prusaslicerthumbnails", filename + ".png"))
```

Comment:
wouldn't it be better to set a internal variable for for the full path and then check it's existence and open it using the variable? Not that it really matters, but would be a couple of less function calls this way.